Makefile

```diff
@@ -97,6 +97,10 @@ KIND_VERSION ?= latest
 MINIKUBE ?= $(LOCALBIN_TOOLING)/minikube
 MINIKUBE_VERSION ?= latest
 
+# The release version of Yunikorn images
+# used in integration and performance tests.
+YK_VERSION=fab384e
```

Comment:
I just have a small question.
So, we have to change this version every time we release a YK image.
Why don't we release with the `latest` tag?

Reply:
I always wonder that!

Reply:
We will change this to `latest` after we unleash our first `v1.6.0-1-gr0` (or whatever) image release, and @pavlovic-ivan can also make a `latest` tag alias to that.